chore(wallet)_: include l1 fees info (for the tx and approval) in the response

saledjenic opened the pull request:
A while ago we disabled l1 info cause the estimated L1 fees were significantly higher than they should have been (if I remember well, the L1 fees were about 1000 times higher than they should have been).

From this point, that wrong value might be coming from unreliable pokt calls/values, these changes bring them back.

saledjenic committed:
chore(wallet)_: include l1 fees info (for the tx and approval) in the response

While ago we've disabled l1 info cause the estimated L1 fees were significantly higher than they should have been
(if I remember well, the L1 fees were about 1000 times higher than they should have been).

From this point, that wrong value might be coming from unreliable pokt calls/values, these changes bring them back.

saledjenic commented:
Initiative for this PR is coming from this comment https://github.com/status-im/status-mobile/pull/21876#issuecomment-2598797177

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (77)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 67b8da33 | [#1](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6271/1/) | 2025-01-20 10:38:45 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250120-103447-67b8da-pr6271.zip) |
| :heavy_check_mark: | 67b8da33 | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6271/1/) | 2025-01-20 10:40:17 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250120-103448-67b8da-pr6271.zip) |
| :heavy_check_mark: | 67b8da33 | [#1](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6271/1/) | 2025-01-20 10:40:27 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250120-103452-67b8da-pr6271.zip) |
| :heavy_check_mark: | 67b8da33 | [#1](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6271/1/) | 2025-01-20 10:40:36 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250120-103446-67b8da-pr6271.aar) |
| :heavy_check_mark: | 67b8da33 | [#1](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6271/1/) | 2025-01-20 10:40:45 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250120-103449-67b8da-pr6271.zip) |
| :heavy_multiplication_x: | 67b8da33 | [#1](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6271/1/) | 2025-01-20 10:40:49 | ~5 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6271/1//consoleText) |
| :heavy_check_mark: | 67b8da33 | [#1](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6271/1/) | 2025-01-20 10:41:04 | ~6 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250120-103454-67b8da-pr6271.zip) |
| :heavy_multiplication_x: | 67b8da33 | [#1](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6271/1/) | 2025-01-20 11:05:34 | ~30 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6271/1//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 214e4250 | [#2](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6271/2/) | 2025-01-23 18:27:23 | ~5 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250123-182135-214e42-pr6271.zip) |
| :heavy_check_mark: | 214e4250 | [#2](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6271/2/) | 2025-01-23 18:59:37 | ~6 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250123-185330-214e42-pr6271.aar) |
| :heavy_multiplication_x: | 214e4250 | [#2](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6271/2/) | 2025-01-23 21:33:14 | ~31 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6271/2//consoleText) |
| :heavy_multiplication_x: | 214e4250 | [#2](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6271/2/) | 2025-01-23 23:20:05 | ~5 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6271/2//consoleText) |
| :heavy_check_mark: | 214e4250 | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6271/2/) | 2025-01-24 10:47:51 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250124-104239-214e42-pr6271.zip) |
| :heavy_check_mark: | 214e4250 | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6271/2/) | 2025-01-24 11:21:03 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250124-111534-214e42-pr6271.zip) |
| :heavy_check_mark: | 214e4250 | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6271/2/) | 2025-01-24 11:26:09 | ~7 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250124-111833-214e42-pr6271.zip) |
| :heavy_check_mark: | 214e4250 | [#2](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6271/2/) | 2025-01-24 13:05:48 | ~7 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250124-125829-214e42-pr6271.zip) |
| :heavy_multiplication_x: | 214e4250 | [#3](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6271/3/) | 2025-01-24 13:05:58 | ~3 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6271/3//consoleText) |
| :heavy_multiplication_x: | 214e4250 | [#4](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6271/4/) | 2025-01-24 13:23:57 | ~3 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6271/4//consoleText) |
| :heavy_multiplication_x: | 214e4250 | [#3](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6271/3/) | 2025-01-24 13:49:36 | ~29 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6271/3//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 39ed1235 | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6271/3/) | 2025-01-24 13:43:59 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250124-134003-39ed12-pr6271.zip) |
| :heavy_check_mark: | 39ed1235 | [#3](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6271/3/) | 2025-01-24 13:44:15 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250124-134001-39ed12-pr6271.zip) |
| :heavy_check_mark: | 39ed1235 | [#3](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6271/3/) | 2025-01-24 13:44:45 | ~4 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250124-134010-39ed12-pr6271.zip) |
| :heavy_check_mark: | 39ed1235 | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6271/3/) | 2025-01-24 13:45:30 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250124-134006-39ed12-pr6271.zip) |
| :heavy_check_mark: | 39ed1235 | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6271/3/) | 2025-01-24 13:45:36 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250124-134002-39ed12-pr6271.zip) |
| :heavy_check_mark: | 39ed1235 | [#3](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6271/3/) | 2025-01-24 13:45:43 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250124-134001-39ed12-pr6271.aar) |
| :heavy_multiplication_x: | 39ed1235 | [#5](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6271/5/) | 2025-01-24 13:45:52 | ~5 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6271/5//consoleText) |
| :heavy_multiplication_x: | 39ed1235 | [#4](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6271/4/) | 2025-01-24 14:20:19 | ~30 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6271/4//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 3a81b49c | [#4](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6271/4/) | 2025-01-24 15:12:49 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250124-150900-3a81b4-pr6271.zip) |
| :heavy_check_mark: | 3a81b49c | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6271/4/) | 2025-01-24 15:12:58 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250124-150901-3a81b4-pr6271.zip) |
| :heavy_check_mark: | 3a81b49c | [#4](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6271/4/) | 2025-01-24 15:13:17 | ~4 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250124-150907-3a81b4-pr6271.zip) |
| :heavy_check_mark: | 3a81b49c | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6271/4/) | 2025-01-24 15:14:26 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250124-150904-3a81b4-pr6271.zip) |
| :heavy_check_mark: | 3a81b49c | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6271/4/) | 2025-01-24 15:14:37 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250124-150901-3a81b4-pr6271.zip) |
| :heavy_multiplication_x: | 3a81b49c | [#6](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6271/6/) | 2025-01-24 15:14:38 | ~5 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6271/6//consoleText) |
| :heavy_check_mark: | 3a81b49c | [#4](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6271/4/) | 2025-01-24 15:14:47 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250124-150858-3a81b4-pr6271.aar) |
| :heavy_check_mark: | 3a81b49c | [#5](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6271/5/) | 2025-01-24 15:40:09 | ~31 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6271/5//consoleText) |
| | | | | | | |
| :heavy_check_mark: | b554ae28 | [#5](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6271/5/) | 2025-01-28 10:10:57 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250128-100704-b554ae-pr6271.zip) |
| :heavy_check_mark: | b554ae28 | [#5](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6271/5/) | 2025-01-28 10:11:40 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250128-100706-b554ae-pr6271.zip) |
| :heavy_check_mark: | b554ae28 | [#5](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6271/5/) | 2025-01-28 10:12:37 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250128-100708-b554ae-pr6271.zip) |
| :heavy_check_mark: | b554ae28 | [#5](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6271/5/) | 2025-01-28 10:12:43 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250128-100703-b554ae-pr6271.aar) |
| :heavy_check_mark: | b554ae28 | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6271/5/) | 2025-01-28 10:12:44 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250128-100704-b554ae-pr6271.zip) |
| :heavy_check_mark: | b554ae28 | [#5](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6271/5/) | 2025-01-28 10:13:03 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250128-100712-b554ae-pr6271.zip) |
| :heavy_check_mark: | b554ae28 | [#7](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6271/7/) | 2025-01-28 10:14:00 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6271/7//consoleText) |
| :heavy_check_mark: | b554ae28 | [#6](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6271/6/) | 2025-01-28 10:37:27 | ~30 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6271/6//consoleText) |
| | | | | | | |
| :heavy_check_mark: | f1265028 | [#6](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6271/6/) | 2025-01-28 23:47:24 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250128-234343-f12650-pr6271.zip) |
| :heavy_check_mark: | f1265028 | [#6](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6271/6/) | 2025-01-28 23:47:56 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250128-234346-f12650-pr6271.zip) |
| :heavy_check_mark: | f1265028 | [#6](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6271/6/) | 2025-01-28 23:48:05 | ~4 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250128-234350-f12650-pr6271.zip) |
| :heavy_check_mark: | f1265028 | [#6](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6271/6/) | 2025-01-28 23:49:20 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250128-234344-f12650-pr6271.zip) |
| :heavy_check_mark: | f1265028 | [#6](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6271/6/) | 2025-01-28 23:49:27 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250128-234342-f12650-pr6271.aar) |
| :heavy_check_mark: | f1265028 | [#6](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6271/6/) | 2025-01-28 23:49:29 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250128-234346-f12650-pr6271.zip) |
| :heavy_multiplication_x: | f1265028 | [#8](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6271/8/) | 2025-01-28 23:50:18 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6271/8//consoleText) |
| :heavy_multiplication_x: | f1265028 | [#7](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6271/7/) | 2025-01-29 00:13:28 | ~29 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6271/7//consoleText) |
| | | | | | | |
| :heavy_check_mark: | c9651fa0 | [#7](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6271/7/) | 2025-01-29 09:49:41 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250129-094550-c9651f-pr6271.zip) |
| :heavy_check_mark: | c9651fa0 | [#7](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6271/7/) | 2025-01-29 09:50:31 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250129-094546-c9651f-pr6271.zip) |
| :heavy_check_mark: | c9651fa0 | [#7](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6271/7/) | 2025-01-29 09:50:40 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250129-094541-c9651f-pr6271.zip) |
| :heavy_check_mark: | c9651fa0 | [#7](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6271/7/) | 2025-01-29 09:51:23 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250129-094546-c9651f-pr6271.zip) |
| :heavy_check_mark: | c9651fa0 | [#7](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6271/7/) | 2025-01-29 09:51:32 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250129-094541-c9651f-pr6271.aar) |
| :heavy_check_mark: | c9651fa0 | [#7](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6271/7/) | 2025-01-29 09:51:38 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250129-094543-c9651f-pr6271.zip) |
| :heavy_multiplication_x: | c9651fa0 | [#9](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6271/9/) | 2025-01-29 09:51:55 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6271/9//consoleText) |
| :heavy_multiplication_x: | c9651fa0 | [#8](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6271/8/) | 2025-01-29 10:15:48 | ~29 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6271/8//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 72ac3ad5 | [#8](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6271/8/) | 2025-01-30 15:59:30 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250130-155526-72ac3a-pr6271.zip) |
| :heavy_check_mark: | 72ac3ad5 | [#8](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6271/8/) | 2025-01-30 15:59:36 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250130-155534-72ac3a-pr6271.zip) |
| :heavy_check_mark: | 72ac3ad5 | [#8](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6271/8/) | 2025-01-30 16:00:02 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250130-155528-72ac3a-pr6271.zip) |
| :heavy_check_mark: | 72ac3ad5 | [#8](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6271/8/) | 2025-01-30 16:00:46 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250130-155530-72ac3a-pr6271.zip) |
| :heavy_check_mark: | 72ac3ad5 | [#8](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6271/8/) | 2025-01-30 16:00:52 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250130-155527-72ac3a-pr6271.zip) |
| :heavy_check_mark: | 72ac3ad5 | [#8](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6271/8/) | 2025-01-30 16:01:16 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250130-155524-72ac3a-pr6271.aar) |
| :heavy_multiplication_x: | 72ac3ad5 | [#10](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6271/10/) | 2025-01-30 16:01:42 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6271/10//consoleText) |
| :heavy_multiplication_x: | 72ac3ad5 | [#9](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6271/9/) | 2025-01-30 16:25:11 | ~29 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6271/9//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 6878af5c | [#9](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6271/9/) | 2025-01-30 19:56:43 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250130-195248-6878af-pr6271.zip) |
| :heavy_check_mark: | 6878af5c | [#9](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6271/9/) | 2025-01-30 19:56:51 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250130-195249-6878af-pr6271.zip) |
| :heavy_check_mark: | 6878af5c | [#9](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6271/9/) | 2025-01-30 19:57:01 | ~4 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250130-195254-6878af-pr6271.zip) |
| :heavy_check_mark: | 6878af5c | [#9](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6271/9/) | 2025-01-30 19:58:24 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250130-195251-6878af-pr6271.zip) |
| :heavy_check_mark: | 6878af5c | [#9](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6271/9/) | 2025-01-30 19:58:47 | ~6 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250130-195245-6878af-pr6271.aar) |
| :heavy_check_mark: | 6878af5c | [#11](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6271/11/) | 2025-01-30 19:59:39 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6271/11//consoleText) |
| :heavy_check_mark: | 6878af5c | [#9](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6271/9/) | 2025-01-30 20:00:21 | ~7 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250130-195249-6878af-pr6271.zip) |
| :heavy_multiplication_x: | 6878af5c | [#12](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6271/12/) | 2025-01-30 20:18:34 | ~4 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6271/12//consoleText) |
| :heavy_multiplication_x: | 6878af5c | [#10](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6271/10/) | 2025-01-30 20:22:07 | ~29 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6271/10//consoleText) |
| :heavy_check_mark: | 6878af5c | [#13](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6271/13/) | 2025-01-30 20:40:42 | ~4 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6271/13//consoleText) |
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | e0ac9200 | [#10](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6271/10/) | 2025-01-30 22:02:51 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250130-215856-e0ac92-pr6271.zip) |
| :heavy_check_mark: | e0ac9200 | [#10](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6271/10/) | 2025-01-30 22:02:57 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250130-215852-e0ac92-pr6271.zip) |
| :heavy_check_mark: | e0ac9200 | [#10](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6271/10/) | 2025-01-30 22:03:28 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250130-215850-e0ac92-pr6271.zip) |
| :heavy_check_mark: | e0ac9200 | [#10](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6271/10/) | 2025-01-30 22:04:21 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250130-215850-e0ac92-pr6271.zip) |
| :heavy_check_mark: | e0ac9200 | [#10](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6271/10/) | 2025-01-30 22:04:40 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250130-215848-e0ac92-pr6271.aar) |
| :heavy_check_mark: | e0ac9200 | [#14](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6271/14/) | 2025-01-30 22:04:58 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6271/14//consoleText) |
| :heavy_check_mark: | e0ac9200 | [#10](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6271/10/) | 2025-01-30 22:05:26 | ~6 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250130-215853-e0ac92-pr6271.zip) |
| :heavy_check_mark: | e0ac9200 | [#11](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6271/11/) | 2025-01-30 22:28:08 | ~29 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6271/11//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 94878433 | [#11](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6271/11/) | 2025-01-31 16:44:19 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250131-164020-948784-pr6271.zip) |
| :heavy_check_mark: | 94878433 | [#11](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6271/11/) | 2025-01-31 16:44:26 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250131-164025-948784-pr6271.zip) |
| :heavy_check_mark: | 94878433 | [#11](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6271/11/) | 2025-01-31 16:44:26 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250131-164027-948784-pr6271.zip) |
| :heavy_check_mark: | 94878433 | [#11](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6271/11/) | 2025-01-31 16:45:45 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250131-164025-948784-pr6271.zip) |
| :heavy_check_mark: | 94878433 | [#11](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6271/11/) | 2025-01-31 16:46:01 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250131-164021-948784-pr6271.zip) |
| :heavy_check_mark: | 94878433 | [#11](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6271/11/) | 2025-01-31 16:46:16 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250131-164019-948784-pr6271.aar) |
| :heavy_check_mark: | 94878433 | [#15](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6271/15/) | 2025-01-31 16:47:14 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6271/15//consoleText) |
| :heavy_check_mark: | 94878433 | [#12](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6271/12/) | 2025-01-31 17:10:12 | ~29 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6271/12//consoleText) |

alaibe reviewed (commented):
Have u confirmed the value makes sense?

saledjenic commented:
> Have u confirmed the value makes sense?

I did not. Expect people from the mobile team to check and respond to that.

pavloburykh commented:
@saledjenic thank you for the PR. Could you please check the [ISSUE 1](https://github.com/status-im/status-mobile/pull/21963#issuecomment-2607625035) identified in corresponding mobile PR?

saledjenic commented:
@pavloburykh could you try again now?

dlipicar reviewed (approved):
changes look good, waiting for confirmation that the values actually make sense

pavloburykh commented:
> @pavloburykh could you try again now?

thank you @saledjenic! @VolodLytvynenko will be able to pick it up tomorrow, as I will be off. Also, I see some builds failed in this PR, is it related to changes of some infra issues?

saledjenic commented:
> is it related to changes of some infra issues?

Should not be up to the changes here, something must be stuck, will push again later and see.

saledjenic commented:
@antdanchenko could you please help on this, cause functional tests must be failing due to missing `OVM_GasPriceOracle` contract on Anvil?

pavloburykh commented:
> @pavloburykh could you try again now?

Thanks @saledjenic. Could you please take a look at the [ISSUE 2](https://github.com/status-im/status-mobile/pull/21963#issuecomment-2612582391) reported by @VolodLytvynenko?

saledjenic commented:
@pavloburykh all that should be already addressed, could you test the mobile app pointing to the most recent changes in this PR?

pavloburykh commented:
> @pavloburykh all that should be already addressed, could you test the mobile app pointing to the most recent changes in this PR?


сс @VolodLytvynenko

saledjenic commented:
I made some comparisons across chains to see how much the values in the Status app differed from the calculations received by Infura's `suggestedGasFees` call that is used by MetaMask.

Here are the findings:
----
Mainnet(1):
```
Status-BaseFee: 	0x1067e80dd (+4403921117WEI) (+4.403921117GWEI) 	Infura-BaseFee: 	3.355443466GWEI
Status-L1 tx fee 	+0WEI (+0GWEI) 
Status-MinPriorityFee: 	0x4b571c0 (+79000000WEI) (+0.079GWEI) 			Infura-MinPriorityFee: 	0.00152421GWEI
Status--------------- 	------- (-------WEI) (-------GWEI) 			Infura-MediumPriorityFee: 	0.625947383GWEI
Status-MaxPriorityFee: 	0x53724e00 (+1400000000WEI) (+1.4GWEI) 			Infura-MaxPriorityFee: 	2GWEI
Status-LowMaxFee: 	0x12b2cb01c (+5019316252WEI) (+5.019316252GWEI) 	Infura-LowMaxFee: 	3.356967676GWEI
Status-MediumMaxFee: 	0x231ab30f9 (+9423237369WEI) (+9.423237369GWEI) 	Infura-MediumMaxFee: 	5.42423154GWEI
Status-HighMaxFee: 	0x33829b1d6 (+13827158486WEI) (+13.827158486GWEI) 	Infura-HighMaxFee: 	9.717519972GWEI

Status-TxEstimatedTime: 		1 (less than minute)
Infura-LOW-WaitTimeEstimate: 		min: 	15sec 	max: 	60sec
Infura-MEDIUM-WaitTimeEstimate: 	min: 	15sec 	max: 	45sec
Infura-HIGH-WaitTimeEstimate: 		min: 	15sec 	max: 	30sec

```
----
Optimism(10):
```
Status-BaseFee: 	0x1ec (+492WEI) (+4.92e-07GWEI) 		Infura-BaseFee: 	3.68e-07GWEI
Status-L1 tx fee 	+318800000000000WEI (+318800GWEI) 
Status-MinPriorityFee: 	0x186a0 (+100000WEI) (+0.0001GWEI) 		Infura-MinPriorityFee: 	0.0001GWEI
Status--------------- 	------- (-------WEI) (-------GWEI) 		Infura-MediumPriorityFee: 	0.0001GWEI
Status-MaxPriorityFee: 	0xf4240 (+1000000WEI) (+0.001GWEI) 		Infura-MaxPriorityFee: 	0.0001GWEI
Status-LowMaxFee: 	0xc0c27 (+789543WEI) (+0.000789543GWEI) 	Infura-LowMaxFee: 	0.000100736GWEI
Status-MediumMaxFee: 	0xc0e13 (+790035WEI) (+0.000790035GWEI) 	Infura-MediumMaxFee: 	0.000101472GWEI
Status-HighMaxFee: 	0xc0fff (+790527WEI) (+0.000790527GWEI) 	Infura-HighMaxFee: 	0.00010368GWEI

Status-TxEstimatedTime: 		1 (less than minute)
Infura-LOW-WaitTimeEstimate: 		min: 	15sec 	max: 	60sec
Infura-MEDIUM-WaitTimeEstimate: 	min: 	15sec 	max: 	45sec
Infura-HIGH-WaitTimeEstimate: 		min: 	15sec 	max: 	30sec
```
----
Arbitrum(42161):
```
Status-BaseFee: 	0xcaf120 (+13300000WEI) (+0.0133GWEI) 		Infura-BaseFee: 	0.01GWEI
Status-L1 tx fee 	+318800000000000WEI (+318800GWEI) 
Status-MinPriorityFee: 	0x0 (+0WEI) (+0GWEI) 				Infura-MinPriorityFee: 	0GWEI
Status--------------- 	------- (-------WEI) (-------GWEI)	 	Infura-MediumPriorityFee: 	0GWEI
Status-MaxPriorityFee: 	0x0 (+0WEI) (+0GWEI) 				Infura-MaxPriorityFee: 	0GWEI
Status-LowMaxFee: 	0xcaf120 (+13300000WEI) (+0.0133GWEI) 		Infura-LowMaxFee: 	0.02GWEI
Status-MediumMaxFee: 	0x195e240 (+26600000WEI) (+0.0266GWEI) 		Infura-MediumMaxFee: 	0.04GWEI
Status-HighMaxFee: 	0x260d360 (+39900000WEI) (+0.0399GWEI) 		Infura-HighMaxFee: 	0.1GWEI

Status-TxEstimatedTime: 		1 (less than minute)
Infura-LOW-WaitTimeEstimate: 		min: 	15sec 	max: 	60sec
Infura-MEDIUM-WaitTimeEstimate: 	min: 	15sec 	max: 	45sec
Infura-HIGH-WaitTimeEstimate: 		min: 	15sec 	max: 	30sec
```
----

saledjenic committed:
chore(wallet)_: adjusting fees calculation to get closer to MM values

saledjenic commented:
Please have a look at the second commit of this PR. I've spent some time adjusting the fees we calculate in the Status app and was trying to make them close to the values displayed in Metamask. Some logic for achieving that is added to the second commit, our fees are slightly higher, but that's ok, I guess.

Logic:
- network congestion is calculated based on priority fee, base fee and gas used ratio
- based on congestion variadic fee is calculated as `networkCongestion x baseFee`
- then low, medium and high fees are calculated as:
  - `lowMaxFees = baseFee + lowPriorityFee`
  - `mediumMaxFees = baseFee + networkCongestion + mediumPriorityFee`
  - `highMaxFees = 2 x baseFee + networkCongestion + lowPriorityFee`

----
Mainnet(1)
```
Status-BaseFee: 	0xda74765f (+3665065567WEI) (+3.665065567GWEI) 		Infura-BaseFee: 	3.599977253GWEI
Status-L1 tx fee 	+0WEI (+0GWEI) 
Status-MinPriorityFee: 	0x5265c0 (+5400000WEI) (+0.0054GWEI) 			Infura-MinPriorityFee: 	0.003102GWEI
Status-MedPriorityFee: 	0x3814fea0 (+940900000WEI) (+0.9409GWEI) 		Infura-MediumPriorityFee: 	0.97GWEI
Status-MaxPriorityFee: 	0x7b1ee888 (+2065623176WEI) (+2.065623176GWEI) 		Infura-MaxPriorityFee: 	2GWEI
Status-LowMaxFee: 	0xdac6dc1f (+3670465567WEI) (+3.670465567GWEI) 		Infura-LowMaxFee: 	3.603079253GWEI
Status-MediumMaxFee: 	0x15f5ffe99 (+5895093913WEI) (+5.895093913GWEI) 	Infura-MediumMaxFee: 	6.117967472GWEI
Status-HighMaxFee: 	0x27cde5ee0 (+10684882656WEI) (+10.684882656GWEI) 	Infura-HighMaxFee: 	10.279947682GWEI

Status-TxEstimatedTime:  		1 (less than minute)
Infura-LOW-WaitTimeEstimate: 		min: 	15sec 	max: 	60sec
Infura-MEDIUM-WaitTimeEstimate: 	min: 	15sec 	max: 	45sec
Infura-HIGH-WaitTimeEstimate: 		min: 	15sec 	max: 	30sec
```
----
Optimism(10)
```
Status-BaseFee: 	0x158 (+344WEI) (+3.44e-07GWEI) 		Infura-BaseFee: 	3.34e-07GWEI
Status-L1 tx fee 	+318800000000000WEI (+318800GWEI) 
Status-MinPriorityFee: 	0x3c (+60WEI) (+6e-08GWEI) 			Infura-MinPriorityFee: 	0.0001GWEI
Status-MedPriorityFee: 	0xf4240 (+1000000WEI) (+0.001GWEI) 		Infura-MediumPriorityFee: 	0.0001GWEI
Status-MaxPriorityFee: 	0x1a76ca (+1734346WEI) (+0.001734346GWEI) 	Infura-MaxPriorityFee: 	0.0001GWEI
Status-LowMaxFee: 	0x194 (+404WEI) (+4.04e-07GWEI) 		Infura-LowMaxFee: 	0.000100668GWEI
Status-MediumMaxFee: 	0xf47a0 (+1001376WEI) (+0.001001376GWEI) 	Infura-MediumMaxFee: 	0.000101336GWEI
Status-HighMaxFee: 	0x1a843a (+1737786WEI) (+0.001737786GWEI) 	Infura-HighMaxFee: 	0.00010334GWEI

Status-TxEstimatedTime: 		1 (less than minute)
Infura-LOW-WaitTimeEstimate: 		min: 	15sec 	max: 	60sec
Infura-MEDIUM-WaitTimeEstimate: 	min: 	15sec 	max: 	45sec
Infura-HIGH-WaitTimeEstimate: 		min: 	15sec 	max: 	30sec
```
----
Arbitrum(42161)
```
Status-BaseFee: 	0x9d2a60 (+10300000WEI) (+0.0103GWEI) 		Infura-BaseFee: 	0.01GWEI
Status-L1 tx fee 	+318800000000000WEI (+318800GWEI) 
Status-MinPriorityFee: 	0x0 (+0WEI) (+0GWEI) 				Infura-MinPriorityFee: 	0GWEI
Status-MedPriorityFee: 	0x0 (+0WEI) (+0GWEI) 				Infura-MediumPriorityFee: 	0GWEI
Status-MaxPriorityFee: 	0x0 (+0WEI) (+0GWEI) 				Infura-MaxPriorityFee: 	0GWEI
Status-LowMaxFee: 	0x9d2a60 (+10300000WEI) (+0.0103GWEI) 		Infura-LowMaxFee: 	0.02GWEI
Status-MediumMaxFee: 	0x274a980 (+41200000WEI) (+0.0412GWEI) 		Infura-MediumMaxFee: 	0.04GWEI
Status-HighMaxFee: 	0x623a7c0 (+103000000WEI) (+0.103GWEI) 		Infura-HighMaxFee: 	0.1GWEI

Status-TxEstimatedTime: 		1 (less than minute)
Infura-LOW-WaitTimeEstimate: 		min: 	15sec 	max: 	60sec
Infura-MEDIUM-WaitTimeEstimate: 	min: 	15sec 	max: 	45sec
Infura-HIGH-WaitTimeEstimate: 		min: 	15sec 	max: 	30sec
```
----

alaibe commented on services/wallet/router/fees/suggested_priority.go:
```diff
@@ -121,18 +125,62 @@ func getEIP1559SuggestedFees(feeHistory *FeeHistory) (lowPriorityFee, avgPriorit
+	avgGasUsedRatio := totalGasUsedRatio / float64(len(feeHistory.GasUsedRatio))
+
+	priorityWeight := 0.7
+	gasUsedWeight := 0.3
```
Maybe const?
Also I guess writing comment about what they are would be useful for future code reader

saledjenic commented:
@all @dlipicar @friofry @briansztamfater could you review the second commit, that should bring us closer to metamask values?

dlipicar commented:
phew! Alright, looks like we're closer.
I'd love too see how their vs. our values evolve over time to see how each of us deal with different levels of network congestion. Not sure if you have already, otherwise perhaps you could write a quick script to fetch/calculate the values once every few minutes so we can plot them and more clearly compare them.

saledjenic committed:
chore(wallet)_: use proxy instead of OVM_GasPriceOracle contract for estimating l1 fee

- A new contract is used for estimating L1 fee.
- New contract's addresses are known only for Optimims and Sepolia Optimimsm.
- Old contract and addresses per chain are kept so far, but not in use anymore.

Comparing to other wallets, seems L1 fee is added to the total fee only for the Optimims chain.
So we're doing the same in this commit, disabling L1 fees for other than the Optimism.
